pom.xml

```diff
@@ -140,6 +140,8 @@
 		<license.projectName>Fiji distribution of ImageJ for the life sciences.</license.projectName>
 		<!-- NB: Deploy releases to the SciJava Maven repository. -->
 		<releaseProfiles>deploy-to-scijava</releaseProfiles>
+		<!-- NB: Use ImgLib2 style code formatting (https://github.com/scijava/scijava-coding-style) -->
+		<scijava.coding-style>imglib2</scijava.coding-style>
```

Comment:
Can we keep this changes out of the PR?

Reply:
Alright, will do. But note this change wouldn't have any consequences on normal builds. Only when you run `mvn formatter:format` or `mvn impsort:sort` the formatting would be applied.

Reply:
It's because I want to cherry-pick it...